server/lorax_server/models/custom_modeling/flash_mistral_modeling.py

```diff
@@ -205,7 +205,7 @@ def _load_gqa(config, prefix: str, weights, head_size):
     if type(weight) is tuple:
         weight, input_scale, weight_scale = weight
 
-    if config.quantize not in ["gptq", "awq", "fp8"]:
+    if config.quantize not in ["gptq", "awq", "fp8", "fp8_kv"]:
```

Comment:
can you make this a utility function so we don't need to reimplement for every model?

Reply:
Done.